source/plugins/go/src/oms.go

```diff
@@ -1151,10 +1209,26 @@ func PostDataHelper(tailPluginRecords []map[interface{}]interface{}) int {
 			if containerID == "" || containsKey(StdoutIgnoreNsSet, k8sNamespace) {
 				continue
 			}
+			if containsKey(SystemNamespaces, k8sNamespace) && containsKey(StdoutIncludeSystemNamespaceSet, k8sNamespace) {
```

Comment:
This check "containsKey(SystemNamespaces, k8sNamespace)"  will be redundant if system namespace allow list check in the tomlparser